db/migrate/20160315014321_create_comments.rb

```diff
@@ -0,0 +1,9 @@
+class CreateComments < ActiveRecord::Migration
+  def change
+    create_table :comments do |t|
+      t.string :timestamp
+      t.string :comment_text
```

Comment:
Same note as your `create_posts` migration: do you need `comment_` in the attribute name?